src/aws_okta_processor/commands/authenticate.py

```diff
@@ -30,7 +30,8 @@
             "--factor": "AWS_OKTA_FACTOR",
             "--silent": "AWS_OKTA_SILENT",
             "--no-okta-cache": "AWS_OKTA_NO_OKTA_CACHE",
-            "--no-aws-cache": "AWS_OKTA_NO_AWS_CACHE"
+            "--no-aws-cache": "AWS_OKTA_NO_AWS_CACHE",
+            "--account": "AWS_ACCOUNT"
```

Comment:
Lets call this "AWS_OKTA_ACCOUNT_ALIAS"

This namespaces it out incase Amazon builds tools using AWS_ACCOUNT

Reply:
agreed. fixed